tests/unit/type/stream-cgroups.tcl

```diff
@@ -124,6 +124,20 @@ start_server {
         }
     }
 
+    test {XPENDING with MKGROUP option to create consumer group} {
+	r XADD mystream_extra * t 1
+	r XADD mystream_extra * s 2
+	catch {[r XPENDING mystream_extra mygroup2 - + 10]} err
+        assert_match {*No consumer group 'mygroup2'*} $err
+
+	set pending [r XPENDING mystream_extra mygroup2 MKGROUP - + 10]
+        assert {[llength $pending] == 0}
+	r XREADGROUP GROUP mygroup2 consumer-1 COUNT 10 STREAMS mystream_extra ">"
+	set pending [r XPENDING mystream_extra mygroup2 - + 10]
+        assert {[llength $pending] == 2}
```

Comment:
the tests still has tabs in them, please change to spaces..

Reply:
Correcetd all intendation in the file.